ADR/adr-0010-namespace-metadata

```diff
@@ -25,6 +25,14 @@ We will apply the following labels to Stone Soup namespaces, to make them easier
 - `stonesoup.redhat.com/namespacetype: "user-workspace-data"` for User workspaces where Applications, Components, and so on are defined
 - `stonesoup.redhat.com/namespacetype: "user-deployments"` for the namespaces where GitOps deploys applications for users
 - `stonesoup.redhat.com/namespacetype: "user-builds"` for the namespaces where the Pipeline Service has PipelineRuns
+- `stonesoup.redhat.com/workspace_id: "b2dece58-a260-11ed-a8fc-0242ac120002"` a unique identifier for the instance of the service (user or workspace UUID)
+- `stonesoup.redhat.com/external_organization: "redhat"` the Red Hat orgId
+- `stonesoup.redhat.com/resource_name: "Stonesoup"` identifier for the type of product (allows tracking multiple variants in the same metric)
+- `stonesoup.redhat.com/account_support: "standard"` Standard, Premium, etc. Must match the value from the SKU
```

Comment:
What SKU?

Reply:
Are the allowable values defined by us (anything we want) or by someone else?  What are the allowable values?

Reply:
> What SKU?

SKU is the identifier for the offering, as recorded in backoffice systems. Some examples I got from swatch team e.g.:
https://offering-manager.corp.redhat.com/offerings/view/MCT4121HR
https://offering-manager.corp.redhat.com/offerings/view/MW02164HR

Reply:
> Are the allowable values defined by us (anything we want) or by someone else? What are the allowable values?

The values are defined by swatch-product-onboarding team and below are the accepted values for these labels
account_support: Standard , Premium, or Self-Support
env_usage: Production, Development/Test, or Disaster Recovery
billing_model: marketplace or standard

Reply:
Parag said we wouldn't have a SKU initially, at least until we start charging for the service.  I'm not sure if we need to get a SKU early just for Segment, or if we can leave this blank.  Can you find out? 

Can you add the allowable values to the PR?

Reply:
@amfred Subscription watch team will be able to ingest the usage data even if we leave values blank. But to start billing, integration with SKU is needed.
Also, they mentioned that even if we are not actively selling a service yet, the SKU can be configured & flagged as a non-sellable so that all the other integrations can be completed.

I have included allowable values of sku_account_support and sku_env_usage beside namespace labels. Do I need to create a separate section to add these values?